client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/multimap/RemoteMultimapCacheImpl.java

```diff
@@ -65,6 +66,15 @@ public RemoteMultimapCacheImpl(RemoteCacheManager rcm, RemoteCache<K, Collection
       this.remoteCacheManager = rcm;
```

Comment:
Lets change this so this method just invokes the other constructor. This way we don't need to duplicate the tracing or assignment of variables.